### PR TITLE
command language serialization

### PR DESCRIPTION
**Pull request by Levi-Armstrong**
This adds command language serialization to xml along with parsing of the generated xml file. This was mainly added to support ros msg serialization. The ros message does not support a good way to serialize the Composite Instruction from what I can determine. Instead it will be serialized as a xml string and sent using ros message as xml string. Need to add some additional test covering all implemented Waypoints and Instructions.

### CODE REVIEW

File: tesseract/tesseract_planning/tesseract_command_language/CMakeLists.txt [hunk 1]
```diff
@@ -35,9 +40,9 @@ install(DIRECTORY include/${PROJECT_NAME}
...
 )
 
-if (TESSERACT_ENABLE_TESTING)
-  enable_testing()
+#if (TESSERACT_ENABLE_TESTING)
+#  enable_testing()
   tesseract_add_run_tests_target()
   add_subdirectory(test)
-endif()
+#endif()
```
Comment (mpowelson) [inline tesseract/tesseract_planning/tesseract_command_language/CMakeLists.txt:47]: remove

File: tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/deserailize.h [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/**
+ * @file deserialize.h
```
Comment (mpowelson) [inline tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/deserailize.h:2]: This file is mispelled.

File: tesseract/tesseract_planning/tesseract_command_language/src/deserialize.cpp [hunk 3]
```diff
@@ -0,0 +1,383 @@
+/**
+ * @file deserialize.h
```
Comment (mpowelson) [inline tesseract/tesseract_planning/tesseract_command_language/src/deserialize.cpp:2]: .cpp

File: tesseract/tesseract_planning/tesseract_command_language/test/serialize_test.cpp [hunk 4]
```diff
@@ -0,0 +1,179 @@
+/**
+ * @file utils_test.cpp
```
Comment (mpowelson) [inline tesseract/tesseract_planning/tesseract_command_language/test/serialize_test.cpp:2]: name + author

File: tesseract/tesseract_planning/tesseract_command_language/CMakeLists.txt (hunk 1 above)
Comment (Levi-Armstrong) [reply tesseract/tesseract_planning/tesseract_command_language/CMakeLists.txt:47]: fixed

File: tesseract/tesseract_planning/tesseract_command_language/test/serialize_test.cpp (hunk 4 above)
Comment (Levi-Armstrong) [reply tesseract/tesseract_planning/tesseract_command_language/test/serialize_test.cpp:2]: fixed

File: tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/deserailize.h (hunk 2 above)
Comment (Levi-Armstrong) [reply tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/deserailize.h]: fixed.

**Pull request merged**